unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetRangeValueParserTest.ts

```diff
@@ -183,5 +183,36 @@ export function DynamicFacetRangeValueParserTest() {
         expect(value).toBe('1999/04/17@00:00:00..2016/06/23@00:00:00inc');
       });
     });
+
+    describe('testing with the currency format (formatting only)', () => {
+      beforeEach(() => {
+        const facet = DynamicFacetRangeTestUtils.createFakeFacet({
+          valueFormat: DynamicFacetRangeValueFormat.currency,
+          currencySymbol: '€',
+          valueSeparator: 'to'
```

Comment:
It's not related to this PR but I am curious if the `valueSeparator` can be easily localized? To we pass the string to the localization function?